packages/opentelemetry-core/src/platform/browser/RandomIdGenerator.ts

```diff
@@ -13,19 +13,17 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import {IdGenerator} from '../../trace/IdGenerator';
+import { IdGenerator } from '../../trace/IdGenerator';
 
 declare type WindowWithMsCrypto = Window & {
```

Comment:
Think this should work without declare. Can you try it?

Reply:
Of course, already committed!